src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -266,37 +272,41 @@ public partial class IPEndPoint : System.Net.EndPoint
         public override System.Net.Sockets.AddressFamily AddressFamily { get { throw null; } }
         public int Port { get { throw null; } set { } }
         public override System.Net.EndPoint Create(System.Net.SocketAddress socketAddress) { throw null; }
-        public override bool Equals(object comparand) { throw null; }
+        public override bool Equals(object? comparand) { throw null; }
         public override int GetHashCode() { throw null; }
         public static System.Net.IPEndPoint Parse(System.ReadOnlySpan<char> s) { throw null; }
         public static System.Net.IPEndPoint Parse(string s) { throw null; }
         public override System.Net.SocketAddress Serialize() { throw null; }
         public override string ToString() { throw null; }
-        public static bool TryParse(System.ReadOnlySpan<char> s, out System.Net.IPEndPoint result) { throw null; }
-        public static bool TryParse(string s, out System.Net.IPEndPoint result) { throw null; }
+        public static bool TryParse(System.ReadOnlySpan<char> s, [System.Diagnostics.CodeAnalysis.NotNullWhen(true)] out System.Net.IPEndPoint result) { throw null; }
+        public static bool TryParse(string s, [System.Diagnostics.CodeAnalysis.NotNullWhen(true)] out System.Net.IPEndPoint result) { throw null; }
     }
     public partial interface IWebProxy
     {
         System.Net.ICredentials Credentials { get; set; }
-        System.Uri GetProxy(System.Uri destination);
-        bool IsBypassed(System.Uri host);
+        System.Uri? GetProxy(System.Uri? destination);
+        bool IsBypassed(System.Uri? host);
```

Comment:
Per https://github.com/dotnet/runtime/pull/31794#issuecomment-589252281, both host and destination parameters here should be `Uri` rather than `Uri?`.  Sorry.

Reply:
Fixed.